README.md

```diff
@@ -0,0 +1,112 @@
+
+# Doctors Appoinments Frontend
```

Comment:
- [ ] In my opinion, the app still requires work.  And I mean styling:

1. The design does not look like the requirement:

![image](https://github.com/LoviMadaan/Appointment-frontend/assets/57421823/0d485ab4-b807-49eb-9cf3-7fa5d6fe085e)


3. In the mobile implementation the sidebar is still getting in the way of the content, overflowing it to the bottom:

![image](https://github.com/LoviMadaan/Appointment-frontend/assets/57421823/633fa0c4-c45d-4d12-8b06-0916157e945e)


Note:

This is a special review, designed to improve your chances in the final evaluation.  I am unable to approve your project in this state, but If you think your project is ready for evaluation, you can go ahead, because everything is optional in this special review, and you don't need my approval to proceed.

Having said that, I highly suggest you implement the changes, because here at Microvers as in the industry, you are being given a set of business requirements for which you need to implement a software solution.

Normally, you'll be given a design, which you'll need to recreate, as required.  This is why you are required to implement it this way. This is what you'll be asked to do in your future job.

If you think it is too hard, kindly remember that you've already completed many CSS projects in order to get to this point, which is the final capstone.  You can do it.